packages/google_sign_in/google_sign_in/README.md

```diff
@@ -66,25 +66,32 @@ To use this plugin, follow the
 
 Add the following import to your Dart code:
 
+<?code-excerpt "example/lib/main.dart (Import)"?>
 ```dart
 import 'package:google_sign_in/google_sign_in.dart';
 ```
```

Comment:
We can just remove from 67 to here; this information is in the installation page instructions.